### PR TITLE
updating caption

### PR DESCRIPTION
**Pull request by VidyaMareddy**
testing vidya

### CODE REVIEW

**Comment by VidyaMareddy** [comment]
testing day 2

**Review by VidyaMareddy — COMMENTED** [review]
done

File: _slides/07-VidyaMareddy.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Hello GitHub
```
Comment (VidyaMareddy) [inline _slides/07-VidyaMareddy.md:6]: Finally I am here

Comment (VidyaMareddy) [inline _slides/07-VidyaMareddy.md:6]: second comment

**Review by anivanadore — APPROVED** [review]

**Pull request merged**